src/BlockEntities/HopperEntity.cpp

```diff
@@ -575,17 +648,45 @@ bool cHopperEntity::MoveItemsToChest(cChunk & a_Chunk, Vector3i a_Coords)
 			// Not the same kind of chest
 			continue;
 		}
-
-		auto chest = static_cast<cChestEntity *>(neighbor->GetBlockEntity(a_Coords + ofs));
+		second = true;
+		auto chest = static_cast<cChestEntity *>(
```

Comment:
```suggestion
		auto SideChest = static_cast<cChestEntity *>(
```

Maybe rename the variable for better readabilty. This is a bit confusing